docs/tutorials/annotation/annotate_points.md

```diff
@@ -72,10 +72,14 @@ def create_label_menu(points_layer, labels):
 
     def label_changed(event):
         """Update the Points layer when the label menu selection changes"""
-        selected_label = event.value
+        try:
```

Comment:
When I run the code `selected_label` is just a string (the label value), so calling `event.value` gives an `AttributeError`. I am not sure if there are cases where this is an `Event` type and calling `.value` is necessary, so I added the try-except.

Reply:
Ah, I think that magicgui was updated to use `psygnal`s rather than VisPy events (@tlambert03?), so that this is now indeed just the value of the thing. So rather than the try/except, just rename the parameter name:

```python
def label_changed(new_label: str):
```

Reply:
Done.